forge-gui/res/cardsfolder/upcoming/ill_timed_explosion.txt

```diff
@@ -1,9 +0,0 @@
-Name:Ill-Timed Explosion
```

Comment:
removed by accident?

Reply:
Coudn't get this one to work

Reply:
it's a problem with the paidList being filled on a sub
you can add RememberDiscarded$ and Remember those instead

Reply:
Hmm, I think i'm still missing something

Reply:
I'll check